.github/workflows/beta.yml

```diff
@@ -0,0 +1,31 @@
+# Builds the proxy on the beta toolchain to help catch Rust regressions before they bite us hit stable.
```

Comment:
"before they bite us hit stable" seems grammatically weird...was one of those supposed to be deleted?